benchmark/data/ci-benchmarkData.json

```diff
@@ -0,0 +1,506 @@
+{
+    "context": {
```

Comment:
Does it mean that we always need to update this benchmark data json? 
What is the flow to update this data?

Reply:
For now, we use his one as a reference.
We may need to update it from time to time if some dramatic improvement or degradation will happens

In theory, we can use the one from `main` beach, but I'm not sure about consistency there.

I suggest to start with a "static" one and we will see how it goes, maybe we will be able to move to "dynamically" generated json from main branch (similar as we do for size comparison)